<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PassportComponentControl</class>
 <widget class="QWidget" name="PassportComponentControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="wellLabel">
       <property name="text">
        <string>Well</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="wellComboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="substanceVerticalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="substanceLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>% substance</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="summarySubstanceLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="substanceSpinBox">
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="wellClusterVerticalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="wellClusterLabel">
       <property name="text">
        <string>Well cluster</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="wellClusterComboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="layerVerticalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="layerLabel">
       <property name="text">
        <string>Object/Layer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="layerComboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_11">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="altitudeLabel">
       <property name="text">
        <string>Altitude</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="altitudeComboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="depthLabel">
       <property name="text">
        <string>Depth MD, m</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="depthSpinBox">
       <property name="minimum">
        <double>-10000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>50000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_14">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="fluidTypeLabel">
       <property name="text">
        <string>Fluid type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fluidTypeLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QVBoxLayout" name="dateVerticalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="dateLabel">
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dateEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_12">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="clusterLabel">
       <property name="text">
        <string>Cluster pumping station</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="clusterComboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_13">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="thicknessLabel">
       <property name="text">
        <string>Stratum/Suit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="stratumLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="6" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_16">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="boosterLabel">
       <property name="text">
        <string>Booster pumping station</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="boosterComboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="intervalLabel">
       <property name="text">
        <string>Perforation interval</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="perforationIntervalSpinBox">
       <property name="minimum">
        <double>-10000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>50000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_17">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="oilGasProdDepLabel">
       <property name="text">
        <string>Oil gas production department</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="oilGasProdDepComboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <property name="spacing">
      <number>2</number>
     </property>
    </layout>
   </item>
   <item row="2" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_15">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="depthTypeLabel">
       <property name="text">
        <string>Depth TVD, m</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="depthTVDSpinBox">
       <property name="minimum">
        <double>-10000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>50000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>layerComboBox</tabstop>
  <tabstop>substanceSpinBox</tabstop>
  <tabstop>wellClusterComboBox</tabstop>
  <tabstop>wellComboBox</tabstop>
  <tabstop>depthSpinBox</tabstop>
  <tabstop>altitudeComboBox</tabstop>
  <tabstop>perforationIntervalSpinBox</tabstop>
  <tabstop>stratumLineEdit</tabstop>
  <tabstop>fluidTypeLineEdit</tabstop>
  <tabstop>clusterComboBox</tabstop>
  <tabstop>boosterComboBox</tabstop>
  <tabstop>oilGasProdDepComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
